src/pom.xml

```diff
@@ -143,21 +143,21 @@
         <!-- Testing -->
         <junit.version>4.12</junit.version>
         <hamcrest.version>1.3</hamcrest.version>
-        <mockito.version>2.23.0</mockito.version>
+        <mockito.version>2.28.2</mockito.version>
         <jsonpath.version>1.2.0</jsonpath.version>
-        <greenmail.version>1.5.6</greenmail.version>
+        <greenmail.version>1.5.10</greenmail.version>
```

Comment:
I somehow remember I tried that before and it lead to problems. We should keep that in mind if we run into mailing trouble down the road.

Reply:
What kind of problems? The test-problem has been fixed within this MR!? In case of auth problems, the test-fix could also help in your project!? As far as i could see in the release notes and MR of greenmail, the main change is that some kind of auth mechanism has been introduced there